src/odb/src/codeGenerator/templates/db.h

```diff
@@ -22,7 +22,11 @@ class {{klass.name}} : public dbObject
   {% endfor %}
   {% for _enum in klass.enums %}
     {% if _enum.public %}
-    enum {{ _enum.name }}{% if "type" in _enum %} :{{ _enum.type }}{% endif %}
+      {% if _enum.class %}
+        enum class {{ _enum.name }}{% if "type" in _enum %} :{{ _enum.type }}{% endif %}
```

Comment:
Would
```
enum {{ class if _enum.class }} {{ _enum.name ...
```
work?  It might reduce the duplication.

Reply:
I changed this to one line with a different syntax, wdyt? Thanks!